manager/integration/tests/test_upgrade.py

```diff
@@ -216,6 +230,55 @@ def test_upgrade(longhorn_upgrade_type,
         write_pod_volume_data(core_api,
                               sspod_info['pod_name'],
                               sspod_info['data'])
+
+    # create custom resources
+    # orphan
+    create_orphaned_directories_on_host(
+        client.by_id_volume(pod_volume_name),
+        ["/var/lib/longhorn"],
+        1)
+    # snapshot and backup
+    backup_stores = get_backupstores()
+    if backup_stores[0] == "s3":
+        set_backupstore_s3(client)
+    elif backup_stores[0] == "nfs":
+        set_backupstore_nfs(client)
+        mount_nfs_backupstore(client)
+    backup_vol_name = "backup-vol"
+    backup_vol = create_and_check_volume(client, backup_vol_name, 2, SIZE)
+    backup_vol.attach(hostId=host_id)
+    backup_vol = wait_for_volume_healthy(client, backup_vol_name)
+    data0 = {'pos': 0, 'len': 2 * BACKUP_BLOCK_SIZE,
+             'content': generate_random_data(2 * BACKUP_BLOCK_SIZE)}
+    create_backup(client, backup_vol_name, data0)
+    # system backup
+    system_backup_name = "test-system-backup"
+    client.create_system_backup(Name=system_backup_name)
+    system_backup_wait_for_state("Ready", system_backup_name, client)
+    # support bundle
+    resp = create_support_bundle(client)
+    node_id = resp['id']
+    name = resp['name']
+    wait_for_support_bundle_state("ReadyForDownload", node_id, name, client)
+    # backing image
+    create_backing_image_with_matching_url(
+        client,
+        BACKING_IMAGE_NAME,
+        BACKING_IMAGE_QCOW2_URL)
+    # recurring job
+    recurring_jobs = {
+        "snapshot1": {
+            "task": "snapshot",
+            "groups": [],
+            "cron": "* * * * *",
+            "retain": 2,
+            "concurrency": 1,
+            "labels": {},
+        }
+    }
+    create_recurring_jobs(client, recurring_jobs)
+    check_recurring_jobs(client, recurring_jobs)
```

Comment:
Could we update any volume in the test with the recurring job?

Reply:
refined